api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2435,6 +2434,10 @@ public boolean hasNext() {
               populate();
```

Comment:
Can this also be changed as suggested in the previous comment?

Reply:
No.  My previous comment explains why